Tests/XcodeProjTests/Project/XcodeProjTests.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import PathKit
+import XCTest
+@testable import XcodeProj
+
+final class XcodeProjIntegrationTests: XCTestCase {
```

Comment:
I've added some tests to bump the coverage of `XcodeProj.swift` and verify my changes were not breaking anything